client/my-sites/types/controller.jsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import ReactDom from 'react-dom';
+import page from 'page';
+
+/**
+ * Internal Dependencies
+ */
+import { getSiteFragment, sectionify } from 'lib/route';
+import { pageView } from 'analytics';
+import { setTitle } from 'lib/screen-title/actions';
+import Types from './main';
+
+export function redirect() {
+	page.redirect( '/posts' );
+}
+
+export function list( context ) {
+	const siteId = getSiteFragment( context.path );
+
+	// [TODO]: Translate title text when settled upon
+	setTitle( 'Types', { siteId: siteId } );
```

Comment:
>Would Post Types be a better title here?
 
Updated to "Custom Post Type" in 341d14b . A future pull request should revise this to set the title depending on the selected post type ("Portfolio Items", etc).